features/location.yml

```diff
@@ -0,0 +1,24 @@
+name: Location
+description: "The `Location` global object represents the current address of the web page loaded in the browser. It makes all component parts of the address accessible and can be manipulated to initiate navigations."
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-location-interface
+# `ancestorOrigins` makes this baseline false but is not required for usage of the feature.
+status:
+  compute_from: api.Location
```

Comment:
OK, now that it works with `window.location`, I think we ought to make a few changes here:

- Use `location` as the entry point in the description (though making this explicitly `window.location` would also be fine) and status computation
- Make the description a little more direct and active ("get the parts" versus "mak[ing] accessible", "navigate" versus initiate navigations).
- Include the term "URL" for findability

```suggestion
description: The `location` global object represents the current page's address. You can use it to get the parts of the address (such as `location.hostname` or `location.pathname`) or navigate to another URL.
spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-location-interface
# `ancestorOrigins` makes this baseline false but is not required for usage of the feature.
status:
  compute_from: api.Window.location
```